app/src/org/commcare/activities/HomeScreenBaseActivity.java

```diff
@@ -392,7 +391,6 @@ protected void showLocaleChangeMenu(final CommCareActivityUIController uiControl
                 Localization.setLocale("default");
             } else {
                 String selectedLocale = localeCodes[position];
-                MainConfigurablePreferences.setCurrentLocale(selectedLocale);
```

Comment:
Is there a reason this has been removed ?

Reply:
@rristovic It seems like the relevant preference key which was being set here is still getting used [here](https://github.com/dimagi/commcare-android/blob/RTLSupport/app/src/org/commcare/CommCareApp.java#L235) to set the default locale for that app.

Reply:
Reverted it back.